test/test_tc_mapper_bugs.cc

```diff
@@ -696,6 +696,25 @@ TEST(LayerNorm, ReferenceBelongsToTwoGroups) {
   atCompl.compile("layernorm", inputs, options);
 }
 
+TEST(Halide2Isl, MinInUpperBound) {
```

Comment:
Looks a bit light on the testing side, could wee see stuff with strides and constants such that ```gcd(x.y)==1``` to shake off a little bit more the different paths added in this PR (in particular ```div```)?
Is there anything we can do to "expect" that we get a combination of ```min``` or ```max``` and ```div```?

Reply:
This patchset intends to hotfix the problem reported in #124 hence the test.

> Is there anything we can do to "expect" that we get a combination of min or max and div?

I'm actually not sure if we can get `max` because we always start tensor subscripts at zero...